services/index.js

```diff
@@ -2,14 +2,9 @@
 
 const glob = require('glob');
 
-// Match modules with the same name as their containing directory.
-// e.g. services/appveyor/appveyor.js
-const serviceRegex = /\/services\/(.*)\/\1\.js$/;
-
 function loadServiceClasses() {
   // New-style services
-  const services = glob.sync(`${__dirname}/**/*.js`)
-    .filter(path => serviceRegex.test(path))
+  const services = glob.sync(`${__dirname}/**/*.service.js`)
```

Comment:
Should this still be filtered to only files under the `/services` directory? If we end up with a file called `something.service.js` in `/lib` or something I think we probably don't want to pull it in here

Reply:
Yea, true, though this is `services/index.js` so `__dirname` here is `/services`.

Reply:
Oh yes, good point. Looks like that call to `.filter()` was not doing anything useful :)

Reply:
Before it was filtering out the testers, and any random helper modules that might have been stashed in `services/`. 😉